tests/suites/cassandra_tls/tls_test.go

```diff
@@ -0,0 +1,214 @@
+package cassandra_tls
```

Comment:
Let's put this test file in `tests/suites/tls_test.go`.

Reply:
I followed kudo-kafka-operator's way of having tests in different packages with it's own folder. If we have the `tls_test.go` and `sanity_test.go` under same package sanity they will share the global vars and functions. We won't get full isolation. like https://github.com/mesosphere/kudo-cassandra-operator/blob/master/tests/suites/sanity_test.go#L22 and https://github.com/mesosphere/kudo-cassandra-operator/blob/8932a7e5f58b52495466ca16265aace4b8b3ca97/tests/suites/cassandra_tls/tls_test.go#L19 will conflict

Reply:
I see. In that case, let's move
- `tests/suites/sanity_test.go` to `tests/suites/sanity/sanity_test.go`
- `tests/suites/cassandra_tls/tls_test.go` to `tests/suites/tls/tls_test.go`.

Reply:
Sure :+1: